commit 7d1f23a0d945a0a6b8e2652d193acc6868731bdb (refs/changes/19/4819/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-09-17T17:51:52+02:00 (1 year, 1 month ago)
    
    TRITON-682 imgapi support for requirements.bootrom
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Approved by: Mike Gerdts <mike.gerdts@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index b3c9129..24f7bfd 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # IMGAPI changelog
 
+## 4.5.2
+
+- TRITON-682 imgapi support for requirements.bootrom
+
 ## 4.5.1
 
 - TRITON-760 Fix imgapi-external-manta-setup and imgapi-manta-setup tooling
diff --git a/package.json b/package.json
index 69dd597..2a427e8 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi",
   "description": "Image API to manage images for SDC 7",
-  "version": "4.5.1",
+  "version": "4.5.2",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -18,7 +18,7 @@
     "forkexec": "1.1.0",
     "glob": "7.0.5",
     "handlebars": "4.0.5",
-    "imgmanifest": "3.0.0",
+    "imgmanifest": "3.1.0",
     "json": "9.0.4",
     "jsprim": "^1.4.0",
     "ldap-filter": "0.3.1",
